english/stapler/stapler.md

```diff
@@ -0,0 +1,5 @@
+Do you have my stapler?
+
+[yes](stapler/yes/yes.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.